openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/ProgressbarWidget.vue

```diff
@@ -21,9 +21,11 @@
 -->
 
 <template>
-  <v-progress-linear height="25" :value="_value" :style="computedStyle">
-    <strong>{{ Math.ceil(_value) }}%</strong>
-  </v-progress-linear>
+  <rux-progress
+    :value="_value"
+    class="progress"
+    :style="computedStyle"
+  ></rux-progress>
```

Comment:
New astro progress bar ... I couldn't get styling to work on the existing vuetify progress bar to even come close to what astro wanted.